packages/whats-new/src/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { Guide } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+import { useLocale } from '@automattic/i18n-utils';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import WhatsNewPage from './whats-new-page';
+
+const WhatsNewGuide = () => {
+	const [ showGuide, setShowGuide ] = useState( true );
+	const [ whatsNewData, setWhatsNewData ] = useState( null );
+	const __ = useI18n().__;
+	const locale = useLocale();
+
+	// Load What's New list on first site load
+	useEffect( () => {
+		const proxiedWpcom = wpcom();
+		proxiedWpcom.request = proxyRequest;
+		proxiedWpcom.req
+			.get( { path: `/whats-new/list?locale=${ locale }`, apiNamespace: 'wpcom/v2' } )
+			.then( ( returnedList ) => {
+				setWhatsNewData( returnedList );
+			} );
+	}, [] );
+
+	const toggleWhatsNew = () => setShowGuide( false );
+
+	return (
+		<>
+			Locale is: { locale }
+			{ whatsNewData && showGuide && (
+				<Guide
+					className="whats-new"
+					contentLabel={ __( "What's New at WordPress.com" ) }
```

Comment:
A note on introducing new strings: we should be careful about what strings are final and what aren't. Usually, when we're not sure if a string is final, we'll not pass it through the localization functions until we've confirmed that.